src/main/java/org/broadinstitute/consent/http/resources/Resource.java

```diff
@@ -159,9 +159,15 @@ private interface ExceptionHandler {
         Resource::unableToExecuteExceptionHandler);
     dispatch.put(PSQLException.class,
         Resource::unableToExecuteExceptionHandler);
-    dispatch.put(SQLSyntaxErrorException.class, e ->
-        errorLoggedExceptionHandler(e,
-            new Error("Database Error", Response.Status.INTERNAL_SERVER_ERROR.getStatusCode())));
+    dispatch.put(SQLSyntaxErrorException.class, e -> {
+      if (e.getMessage().contains("invalid byte sequence")) {
+        return errorLoggedExceptionHandler(e,
+            new Error("Invalid byte sequence", Response.Status.BAD_REQUEST.getStatusCode()));
```

Comment:
How can we test this specific error?

Reply:
In any resource test, if you mock the underlying service method to throw the exception, that should exercise this condition.